src/polycubed/src/polycubed.cpp

```diff
@@ -278,6 +284,22 @@ int main(int argc, char *argv[]) {
   // register services that are shipped with polycube
   load_services(*core);
 
+  // create a cubes dump instance if needed
+  if (!config.getCubesNoDump()) {
+    cubesdump = new CubesDump();
+    core->set_cubes_dump(cubesdump);
+  }
+
+  // load the last topology created
+  if (!config.getCubesInitTopology()) {
```

Comment:
Replace the comment with
// In case the user does not want to initialize the Polycube virtual network, let's load the last topology that was present when the daemon was shut down.